unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/Controllers/BIWPublishController.cs

```diff
@@ -81,10 +81,10 @@ public bool CanPublish()
         return true;
     }
 
-    void CheckPublishConditions()
+    public string CheckPublishConditions()
```

Comment:
```suggestion
    public string CheckPublishConditions()
```
Please add some xml docs of what's expected of this return value.